src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -62,6 +62,23 @@
  * <pre>
  * &lt;module name=&quot;WhitespaceAfter&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class MyTest {
+ *  public static void myTest() {
+ *      if (foo) {              // OK
+ *              //...
+ *      } else if(bar) {        // violation
+ *              //...
+ *      }
+ *
+ *      testMethod(foo, bar);   // OK
+ *      testMethod(foo,bar);    // violation
+ *
+ *      for(;;){}               // OK
```

Comment:
Do you know why this is ok ? If this is big it it is better to not put it in example.

Reply:
This is the intended behavior: 

> Checks that a token is followed by whitespace, with the exception that it does not check for whitespace after the semicolon of an empty for iterator. Use Check EmptyForIteratorPad to validate empty for iterators.

So, I thought it best to show that.